spec/rack/test_spec.rb

```diff
@@ -613,6 +613,24 @@ def verb
     def verb
       'delete'
     end
+
+    it 'uses the provided params hash' do
+      delete '/', foo: 'bar'
+      expect(last_request.GET).to eq({})
+      expect(last_request.POST).to eq('foo' => 'bar')
```

Comment:
For the casual reader, it might not be entirely obvious that the body here is _not_ JSON but instead multipart/form-encoded data. Would you mind adding a note about that, or a separate test that tests the content of `last_request.body.read`?